lib/scrape.js

```diff
@@ -95,6 +95,14 @@ async function fetchOrgsWithData() {
   )
 }
 
+async function fetchDates() {
+  const res = await fetch(
+    'https://codein.withgoogle.com/api/program/current/'
+  )
+  const { result } = await res.json()
+  return result
```

Comment:
This fails in CI, there's no `result` key in the json.
Just do `return res.json()`